### DISCUSSION TITLE
Anything ELOT

### DISCUSSION THREAD

**Discussion by johanwk** [johanwk/elot#22] [General]
Ontology editing with Emacs. Express your wishes.

**Comment by johanwk** [comment]
@VladimirAlexiev, @mhodki, I think the basic functionality is in place now. We need to do some basic testing, but it makes sense to try to recruit a couple more people to the project. Maybe we could announce ELOT to the team making ROBOT, what do you think, is it too early?

**Comment by VladimirAlexiev** [comment]
It is not too early, but how many of these people use Emacs? I think you should focus on:
- more IDO WG members
- emacs users that use some semantic tools. But I don't know how to find them... I searched "emacs sparql" on github..
  -  This is curious, but maybe generated by LLM? https://github.com/meta-introspector/ctuning-submodules/issues/1#issuecomment-1789490004. It talks of a `semantic-web` package but I don't think such exists

**Comment by johanwk** [comment]
Unfortunately, that "semantic web" package has all the signs of being generated by AI. There is no such package in the regular Emacs repositories.

**Comment by johanwk** [comment]
There's a bit of a challenge with recruiting IDO WG members these days, as we don't have weekly meetings during the ballot period.

I think I'll try first to get the "label-display" functionality in place, then start thinking more actively about spreading the word. I'm working on an ontology for a work project now, and with uninformative identifiers, the need to be able to switch to seeing labels is very strong. It's too much of a strain on the brain to have to look up names all the time. But once that's in place, I think we can say the tool is more or less ready for use, even though there are many bugs and not yet implemented features.

**Comment by johanwk** [comment]
I've just added a new feature: displaying the `rdfs:label` instead of the, often unreadable, identifier.

Check out the screenshot at the bottom of the README.
